db/write_batch.cc

```diff
@@ -414,8 +414,12 @@ Status WriteBatch::Iterate(Handler* handler) const {
   char tag = 0;
   uint32_t column_family = 0;  // default
   bool last_was_try_again = false;
-  while (((s.ok() && !input.empty()) || UNLIKELY(s.IsTryAgain())) &&
-         handler->Continue()) {
+  bool handlerContinue = true;
+  while (((s.ok() && !input.empty()) || UNLIKELY(s.IsTryAgain()))) {
+    if (!(handlerContinue = handler->Continue())) {
```

Comment:
nit: separate assignment from conditional expression. Windows build (Appveyor) complaining about it.